src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Feedback, FeedbackType, MANUAL_ASSESSMENT_IDENTIFIER } from 'app/entities/feedback.model';
+import { cloneDeep } from 'lodash';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+    selector: 'jhi-code-editor-tutor-assessment-inline-feedback',
+    templateUrl: './code-editor-tutor-assessment-inline-feedback.component.html',
+})
+export class CodeEditorTutorAssessmentInlineFeedbackComponent {
+    MANUAL = FeedbackType.MANUAL;
+    @Input()
+    get feedback(): Feedback {
+        return this._feedback;
+    }
+    set feedback(feedback: Feedback) {
+        this._feedback = feedback || new Feedback();
```

Comment:
I don't think that we use the _ notation so far

Reply:
see comment above :) I just checked and we have it the same way e.g. for drag and drop and short answer questions